### PR TITLE
#14147. Avoid destroy rtcModule::Session when receive IceDisconnection 

### PR DESCRIPTION
**Pull request by alber2510**

### CODE REVIEW

**Comment by alber2510** [comment]
trigger compilation

File: src/rtcModule/webrtc.cpp [hunk 1]
```diff
@@ -1804,7 +1816,9 @@ void Call::removeSession(Session& sess, TermCode reason)
         if (wptr.deleted())
             return;
 
+
```
[the remark this answers is not in the document]
Comment (alber2510) [reply src/rtcModule/webrtc.cpp]: Fixed

File: src/rtcModule/rtcStats.h [hunk 2]
```diff
@@ -58,6 +62,10 @@ class RtcStats: public IRefCountedMixin<IRtcStats>
     std::string mDeviceInfo;
     std::vector<Sample*> mSamples;
     ConnInfo mConnInfo;
+    unsigned long mMaxIceDisconnectionTime = 0;
+    unsigned int mIceDisconnections = 0;
+    karere::Id mPreviousSessioId;
```
[the remark this answers is not in the document]
Comment (alber2510) [reply src/rtcModule/rtcStats.h]: Fixed

**Comment by alxvasilev** [comment]
Unfortunately my review is not accepted by github.

**Comment by alber2510** [comment]
Many thanks for your review @alxvasilev :)

**Pull request merged**